### ISSUE TITLE
pngquant failed to build, make sure that libpng-dev is installed

### ISSUE THREAD

**Issue by odai-alali** [quasarframework/app-extension-icon-factory#68]
When trying installing dependencies on another dev machine (I cloned the app repo and then `npm install`). during the postinstall hooks I get the error 
```
pngquant failed to build, make sure that libpng-dev is installed
```

On the first machine, the extension worked without problem and I didn't have to install any png library manually. Both machines are Windows 10.

**Comment by nothingismagick** [comment]
Use yarn, not npm.

>

**Comment by odai-alali** [comment]
I used npm on the first machine and it worked without problems. Anyway, I tried using yarn and it didn't help.

**Comment by nothingismagick** [comment]
Please post the results of

quasar info

**Comment by odai-alali** [comment]
Will do as soon as I have access to the machine with the problem. Thank you for the very fast reply.

**Comment by odai-alali** [comment]
Hi, that's what I got

```
Operating System                Windows_NT(10.0.17134) - win32/x64
NodeJs                          8.12.0

Global packages
 NPM                           6.4.1
 yarn                          1.16.0
 quasar-cli                    0.17.25
 vue-cli                       Not installed
 cordova                       9.0.0 (cordova-lib@9.0.1)

Important local packages
 quasar-cli                    Not installed
 quasar-framework              Not installed
 quasar-extras                 Not installed
 vue                           2.6.10  (Reactive, component-oriented view layer for modern web interfaces.)
 vue-router                    3.0.6   (Official router for Vue.js 2)
 vuex                          3.1.1   (state management for Vue.js)
 electron                      Not installed
 electron-packager             Not installed
 electron-builder              Not installed
 @babel/core                   7.4.5   (Babel compiler core.)
 webpack                       4.32.2  (Packs CommonJs/AMD modules for the browser. Allows to split your codebase into multiple bundles, which can be loaded on demand. Support loaders to preprocess files, i.e. json, jsx, es7, css, less, ... and your custom stuff.)
 webpack-dev-server            3.3.1   (Serves a webpack app. Updates the browser on changes.)
 workbox-webpack-plugin        4.3.1   (A plugin for your Webpack build process, helping you generate a manifest of local files that workbox-sw should precache.)
 register-service-worker       1.6.2   (Script for registering service worker, with hooks)

Networking
 Host                          DESKTOP-5NN1D8H
 Ethernet                      192.168.0.158

´´´
Operating System                Windows_NT(10.0.17134) - win32/x64
NodeJs                          8.12.0

Global packages
 NPM                           6.4.1
 yarn                          1.16.0
 quasar-cli                    0.17.25
 vue-cli                       Not installed
 cordova                       9.0.0 (cordova-lib@9.0.1)

Important local packages
 quasar-cli                    Not installed
 quasar-framework              Not installed
 quasar-extras                 Not installed
 vue                           2.6.10  (Reactive, component-oriented view layer for modern web interfaces.)
 vue-router                    3.0.6   (Official router for Vue.js 2)
 vuex                          3.1.1   (state management for Vue.js)
 electron                      Not installed
 electron-packager             Not installed
 electron-builder              Not installed
 @babel/core                   7.4.5   (Babel compiler core.)
 webpack                       4.32.2  (Packs CommonJs/AMD modules for the browser. Allows to split your codebase into multiple bundles, which can be loaded on demand. Support loaders to preprocess files, i.e. json, jsx, es7, css, less, ... and your custom stuff.)
 webpack-dev-server            3.3.1   (Serves a webpack app. Updates the browser on changes.)
 workbox-webpack-plugin        4.3.1   (A plugin for your Webpack build process, helping you generate a manifest of local files that workbox-sw should precache.)
 register-service-worker       1.6.2   (Script for registering service worker, with hooks)

Networking
 Host                          DESKTOP-5NN1D8H
 Ethernet                      192.168.0.158
```

**Comment by nothingismagick** [comment]
This is only available for quasar 1.0

>

**Comment by odai-alali** [comment]
🙈 how did I missed that... Thanks
